packages/compass-query-bar/src/utils/query.ts

```diff
@@ -51,6 +51,35 @@ export function doesQueryHaveExtraOptionsSet(fields?: QueryFormFields) {
   return false;
 }
 
+export function parseQueryStringsToFormFields(query?: {
```

Comment:
`parseQueryAttributesToFormFields` or `parseQueryPropertiesToFormFields`? .. maybe is just me but when i read  parseQueryStrings i can only think that we are parsing a URL